### PR TITLE
fix: use isnil to validate callback function

### PR DESCRIPTION
**Pull request by slavik-lvovsky**

### CODE REVIEW

File: backend/src/youi-adapter.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export class YouiAdapter extends TerminalAdapter {
   ): Promise<T2> {
     if (this.yeomanui && questions) {
       const result: any = await (this.yeomanui.showPrompt(questions) as Promise<T2>);
-      if (!_.isEmpty(cb)) {
+      if (!_.isNil(cb)) {
```
Comment (tomer-epstein) [inline backend/src/youi-adapter.ts:48]: I think we should also update/add test for param cb=null

File: backend/tests/youi-adapter.spec.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    describe('#prompt()', () => {
+        it('passes null call back', async () => {
+            const youiAdapter = new YouiAdapter(youiEvents, outputChannel); 
+            youiAdapter.setYeomanUI(yeomanUi);
+            const response = await youiAdapter.prompt([], null);
+            expect(response).to.be.empty;
+        });
+    });
+});
```
Comment (tomer-epstein) [inline backend/tests/youi-adapter.spec.ts:90]: it's doing the job. but can we test not empty question obj ?
const questions = [{ name: "q1" }];

rpc.invoke can return --> 
{
					firstName: "john",
					lastName: "doe"
				};

File: backend/src/youi-adapter.ts (hunk 1 above)
Comment (slavik-lvovsky) [reply backend/src/youi-adapter.ts]: When use isFunction cb === null is already checked

**Review by tomer-epstein — APPROVED** [review]
